provision/start_chain.sh

```diff
@@ -1,13 +1,13 @@
 #/bin/sh
 
-# read no.of nodes to be setup
+# read no.of nodes to be setup passed as the first argument
 NODES=$1
 if [ -z $NODES ]
 then
     NODES=2
 fi
 
-# read no of accounts to be setup
+# read no of accounts to be setup passed as the second argument
 ACCOUNTS=$2
 echo " ** Number of nodes : $NODES and accounts : $ACCOUNTS to be setup **"
```

Comment:
```suggestion
echo " INFO: Setting up $NUM_NODES validator nodes and $NUM_ACCOUNTS accounts"
```

Remove all the extra `****` in the logs